src/agoradatatools/great_expectations/gx/expectations/gene_info.json

```diff
@@ -514,25 +503,25 @@
       "kwargs": {
         "column": "biodomains",
         "list_members": [
-          "Apoptosis",
-          "Vasculature",
-          "Lipid Metabolism",
           "Oxidative Stress",
-          "Mitochondrial Metabolism",
+          "Synapse",
```

Comment:
This is so weird. I just ran the notebook on my machine and got a _third_ different ordering. I know Python lists aren't guaranteed to be in the order you put them but... wow.

Reply:
@BWMac Update: Putting the values in alphabetical order in the ipynb (or calling sorted() on the original list) seems to get them to stay in alphabetical order in the json output. So maybe we should just be doing that, assuming this works on your machine too.

Reply:
ah nice! That does work for me too. Kinda annoying but I suppose it isn't too much of a pain to do this. Thanks @jaclynbeck-sage !

Reply:
@beatrizsaldana would you mind adding this change to the gene_info ipynb in your PR so we don't have commits with list order issues after this PR? I think it would be good to alphabetize the possible values for pharos class too, while you're there.

Reply:
Done!